### DISCUSSION TITLE
IIP - hitchhiker staking with virtual identity

### DISCUSSION THREAD

**Discussion by ubiubi18** [idena-network/idena-docs#178] [IIPs]
This is an idea how to introduce a virtual identity to idenanetwork, that allows staking for unvalidated identities https://github.com/ubiubi18/HitchhikerStaking/blob/main/README.md

**Comment by morienzo** [comment]
I find this mechanism very interesting; it would open the door to investors who do not wish to participate in securing the network. I even think it could already be implemented by a third party through a dapp (the process would be somewhat different, of course).

**Comment by reb0rn21** [comment]
I also agree, with this we might open a door a but for investors 
I do not see majority nor most investors validating ever

**Comment by 0JustinMiles0** [comment]
The concept of hitchhiker staking for validated identities who don't want to run a node makes sense but I don't understand the value of allowing non-validated identities to earn inflation rewards. So basically, people could earn inflationary rewards by contributing nothing to the security of the network? It seems to me that it will skew the incentives on which the network is built. I don't really see what's in for those who contribute to the security of the network. They would just be giving away a share of the pie and it would make Idena no more different than a regular blockchain.

**Reply by ubiubi18** [reply]
See https://github.com/idena-network/idena-docs/discussions/178#discussioncomment-9322527

**Comment by reb0rn21** [comment]
You are right about that, but also validation is so complex process that 99.9% investors will not lose time on that
Is the main reason idena failed as mass identity network as whole concept is way to time consuming, complex for majority of the ppl

But this is just one problem, 2nd is infinite inflation that do not track network usage nor network size

**Reply by ubiubi18** [reply]
The moment you cap the supply, you need to introduce tax or kill the long term perspective. But that would be a different IIP anyway?

**Comment by hlolve** [comment]
Why do you call " Hitchhiker" ? I see just a way to give minting rewards for hodlers. They will not validate and will not run a node. The only risk is to have idnas, and not locked,  but at a cost of 0,5% withdraw 

It will reduce mining rewards (in idnas) for all that secure the network (running nodes and making flips and validating). The positive part could be a potential increase in price.

"Reward Distribution: 50% of rewards to stakeholders, 50% to zerowallet" Zerowallet has its participation in validation rewards, I don't see the point to include here

**Reply by ubiubi18** [reply]
Yes, APY for validated identities would come down, but virtual identities APY would come down faster.  If the price would keep growing with the APY declining, it could feel less harmful. Look at ethereums APY. They have only around 4%, since they are established and well connected with traditional financial market incentives. 

If idena was that established, the virtual identity APY would come down below 4%, while the identity stakers APY would stay above 4%. This sounds irrelevant, but in the long term that would be the game changer that could canibalize a lot in favor of idena

**Reply by ubiubi18** [reply]
It is important to understand it in relative terms to marketcap and not in absolute terms to coin supply.

**Reply by 0JustinMiles0** [reply]
Quadratic/Identity staking is the core value proposition of the Idena blockchain that justifies to be invested in. Why would investors put money in Idena if we remove these properties? There are dozen of more performant blockchains out there for investors to choose from. This proposal changes the entire incentive structure of the blockchain and investment thesis. If that were to be implemented, I would pull out my investment from Idena.

**Reply by 0JustinMiles0** [reply]
Also, I don't understand why the proposal is called "virtual identity" since the concept of "virtual identity" seems to be no more different than the concept of "wallet address". The proposal doesn't even touch on the impact of the quadratic distribution in a an environment where one single investor could spread his stake across hundreds of addresses. In short, I don't get what this proposal aims to fix/solve while it would certainly introduce a lot of issues/unknown.

**Reply by ubiubi18** [reply]
No, the proposal includes just one virtual identity, which competes with all other identities in quadratic staking. Mikhail has outlined the idea in the chronicles that are linked in the proposal. There is no linear staking and you cant set up hundreds of virtual identities. But hundreds of investors could delegate their IDNA to the same virtual identity, bringing down its APY that way. Aditionally it doesnt even earn the full APY.

**Reply by 0JustinMiles0** [reply]
Gotcha, I misunderstood the proposal then. I find the idea worth exploring. Do I understand correctly that mining rewards would be earned by the pool at every block? - "The rewards are generated continously as virtual reward drawing rights with every block".

If so, then this reward should be no more than if the identity was a validated identity meaning that it should decrease as the number of validated identities increase. I would not support this proposal if it was a fix amount (regardless of how many validated identities mine) taken out of the block reward on evert block.

**Comment by LL-10** [comment]
Idena stake is reserved for validated unique identities for some reason

**Comment by Reddmore** [comment]
I like the current conditions more.

**Comment by reb0rn21** [comment]
Then you most like idena going to 0$ and delist one day....
idena refused to make big change when it seen it failed its main goal some 3y ago, I see few users that still refuse to accept the reality. so then be it
If idena refuse to adopt to the reality and stick to failed goal I see no hope here

You all first need to understand what you or me like and what is possible and real is not same, sadly most ppl are not ready to understand it
I alone hate staking but in some cases as for idena it can have logic, but also making idena work only for the ones which are insane to invest a lot time and life in validation is pure insanity as 99.99999% are not ready to do constant validation and never will be in this form

**Comment by morienzo** [comment]
Wen IIP?

**Reply by ubiubi18** [reply]
Vote is still counting, and also we got no feedback from devs yet if it is worth to consider such ideas @ozneirom
